frontend/src/main/components/Sections/SectionsTable.js

```diff
@@ -1,67 +1,109 @@
-import OurTable from "main/components/OurTable";
+import SectionsTableBase from "main/components/SectionsTableBase";
 
 import { yyyyqToQyy } from "main/utils/quarterUtilities.js";
-import { convertToFraction, formatDays, formatInstructors, formatLocation, formatTime, isSectionAsString } from "main/utils/sectionUtils.js";
+import { convertToFraction, formatDays, formatInstructors, formatLocation, formatTime, isSection, _isSectionAsString} from "main/utils/sectionUtils.js";
+
+
+function getFirstVal(values) {
+    return values[0];
+};
 
 export default function SectionsTable({ sections }) {
 
 
     // Stryker enable all 
-
+    // Stryker disable BooleanLiteral
     const columns = [
         {
             Header: 'Quarter',
```

Comment:
In my opinion, I think a quarter column is unnecessary, since all searches / schedules are already restricted to a given quarter anyway. I would suggest either omitting this column or taking @pconrad's suggestion of putting the section number (e.g. `0100` or `0201`) here instead.

Reply:
I'd prefer to keep quarter, since it will be needed eventually for searches that span multiple quarters, and this is a multipurpose component.

If we don't want it to show up, I'd prefer to add a prop like this:

```
export default function SectionsTable({ sections, hideQuarter=false })
```

And then use the value of `hideQuarter` to either include or exclude the quarter column.